libs/nss.c

```diff
@@ -100,11 +109,8 @@ int _nss_tcb_getspent_r(struct spwd *__result_buf,
 	off_t currpos;
 	int retval, saved_errno;
 
-	if (!tcbdir) {
-		tcbdir = opendir(TCB_DIR);
-		if (!tcbdir)
-			return NSS_STATUS_UNAVAIL;
-	}
+	if (!tcbdir)
+		return NSS_STATUS_UNAVAIL;
```

Comment:
Should we set `errno`? Previously, it'd be set by the `opendir` call here.

Reply:
How about `errno = ENOENT;`?

Reply:
Yes, I think `ENOENT` is our best option here.

Reply:
> Yes, I think `ENOENT` is our best option here.

Added.